v1/csharp-starter/nitric.yaml

```
@@ -0,0 +1,10 @@
+name: csharp-starter
+services:
+  - match: services/**/*.csproj
+    start: dotnet run --project $SERVICE_PATH
```

```suggestion
    start: dotnet watch --project $SERVICE_PATH
```

Should this be watch? I couldn't get hot reloading working without this change.